Core/Core/Offline/Model/OfflineModeInteractor.swift

```diff
@@ -23,11 +23,18 @@ import CoreData
 public protocol OfflineModeInteractor {
     func isFeatureFlagEnabled() -> Bool
     func isOfflineModeEnabled() -> Bool
+    func isNetworkOffline() -> Bool
     func observeIsFeatureFlagEnabled() -> AnyPublisher<Bool, Never>
     func observeIsOfflineMode() -> AnyPublisher<Bool, Never>
     func observeNetworkStatus() -> AnyPublisher<NetworkAvailabilityStatus, Never>
 }
 
+public extension OfflineModeInteractor {
+    func isNetworkOffline() -> Bool {
+        false
+    }
+}
+
```

Comment:
I know protocol default implementations are used in the code frequently but we should avoid them. See [this article](https://medium.com/@xcadaverx/default-protocol-extension-caveats-88c6ece27fdb) why this can become an issue.

Reply:
I think we can safely remove this extension now.

Reply:
oops, I did, just got lost during branch switches